### DISCUSSION TITLE
Navigation in Qfield

### DISCUSSION THREAD

**Discussion by diegofhernandez98** [opengisch/QField#4122] [Q&A]
Hello, im using Qfield to navigate to certain locations in the field. however, i found that the setting location as destination is showing a straight line rather than proper directions. Can anyone explain me if this is common or is an error ?
![issue](https://user-images.githubusercontent.com/129914865/229956578-932bc4eb-e5ca-4663-bd6d-214e60ab7e9f.png)

**Comment by nirvn** [comment]
@diegofhernandez98 , currently, QField's navigation has only one "fly to destination" mode that shows a straight line between your current location and destination. What you are looking for is routing functionality, which isn't in QField yet. It would probably require some form of crowdfunding for it to happen.
